docs/variables.md

```diff
@@ -6,61 +6,64 @@ title: Variables References
 
 ### User directories
 
-| Description | Name | Default Value |
+| Description | Name | Default Value(s) |
 |-------------|:----:|---------------|
+| Books | `@{XDG_BOOKS_DIR}` | `Books` |
 | Desktop | `@{XDG_DESKTOP_DIR}` | `Desktop` |
-| Download | `@{XDG_DOWNLOAD_DIR}` | `Downloads` |
-| Templates | `@{XDG_TEMPLATES_DIR}` | `Templates` |
-| Public | `@{XDG_PUBLICSHARE_DIR}` | `Public` |
+| Disk images | `@{XDG_IMG_DIR}` | `images` |
```

Comment:
I just found this after re-reading my changes, was this lowercase intentionally? The rest of the options start with a capital.

Reply:
Yes, it was as it is not really a "media" like file.

Reply:
As this is not straightforward, maybe we should change the directory name to more verbose one.